code/frontend/src/util/SpeechToText.tsx

```diff
@@ -0,0 +1,22 @@
+import {
+    SpeechConfig,
+    AudioConfig,
+    SpeechRecognizer,
+    AutoDetectSourceLanguageConfig,
+  } from "microsoft-cognitiveservices-speech-sdk";
+
+
+export const multiLingualSpeechRecognizer = (
+    subscriptionKey: string,
+    serviceRegion: string,
+    languages: string[]
+  ) => {
+        const speechConfig = SpeechConfig.fromSubscription(
+          subscriptionKey,
+          serviceRegion
+        );
+        const audioConfig = AudioConfig.fromDefaultMicrophoneInput();
+        var autoDetectSourceLanguageConfig = AutoDetectSourceLanguageConfig.fromLanguages(languages)
```

Comment:
Nit: Is there any reason this is a `var` and the rest are `const`? Probably doesn't matter either way, just noticed it was different!

Reply:
Oops, missed it when changing the others to const. Will edit this, thanks!